hbook/ntup/ntup_example_test.go

```diff
@@ -47,3 +48,67 @@ func ExampleNtuple_scanH2D() {
 	// XStdErr:    0.957427
 	// YStdErr:    0.957427
 }
+
+func ExampleNtuple_scanH() {
+	nt, err := ntcsv.Open(
+		"ntcsv/testdata/simple-with-header.csv",
+		ntcsv.Comma(';'),
+		ntcsv.Header(),
+		ntcsv.Columns("v1", "v2", "v3"),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer nt.DB().Close()
+	var (
+		xmin = +math.MaxFloat64
+		xmax = -math.MaxFloat64
+		ymin = +math.MaxFloat64
+		ymax = -math.MaxFloat64
+	)
+	query := "v1, v2"
+	error_ := nt.Scan(query, func(x, y float64) error {
+		xmin = math.Min(xmin, x)
+		xmax = math.Max(xmax, x)
+		ymin = math.Min(ymin, y)
+		ymax = math.Max(ymax, y)
+		return nil
+	})
+	if error_ != nil {
+		log.Fatal(error_)
+	}
+
+	fmt.Printf("Result  %v", error_)
```

Comment:
instead of printing a "nil" result, why don't we print the min/max of v1 and v2?